sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -484,6 +490,7 @@ func componentsToResourceData(components []*cloudauth.AccountComponent) []map[st
 		resourceData := map[string]interface{}{}
 		resourceData[SchemaType] = component.GetType().String()
 		resourceData[SchemaInstance] = component.GetInstance()
+		resourceData[SchemaVersion] = component.GetVersion()
```

Comment:
Shall we update the ACC tests as well once BE PR is merged?

Reply:
I don't think this actually depends on new BE functionality. We can add the field in tests now.

Reply:
done